src/components/table/makeSelectable.js

```diff
@@ -613,7 +609,12 @@ function makeSelectable(BaseTable) {
         isTargetSlider = event => event.target?.role === 'slider';
 
         // Workaround for focus conflicting with Blueprint components in Search Quick Filters, not needed once Blueprint table is integrated
```

Comment:
Can we update this comment, maybe put the comments directly above each case in `isTargetQuickSearch`